### PR TITLE
Minor fixes of plots

### PR DESCRIPTION
**Pull request by yerbol-akhmetov**
Good day. Here I propose several changes.

Plots:
- [x] **Transmission congestion plot**: reorder lines and links to avoid green on top of black
- [x] **Electricity generation plot**: merge coal and lignite
- [x] **Electricity bills plot**: expand limits for prices and bills, 
- [x]  **Electricity bills plot**: calculate price using electricity consumption only
- [x] **Curtailment plot**: update location and transparency of legend
- [x] **Hydrogen production plot**: update location and transparency of legend
- [x] **Heat tech ratio plot**: update limits
- [x] **CO2 levels plot**: add title for plots
- [x] **Capacity expansion plot**: add a script
- [x] **RES share plot**: add a script
- [x] **Electricity for heat plot**: fix plot for BAU
- [x] **Total costs plot**: adjust limits
- [x] **Total costs plot**: move functions and constants into `_helpers.py`
- [x] **Capacity expansion plot**: move functions and constants into `_helpers.py`
- [x] Update `config.plot.yaml` for dist2.0

Run.py:
- [x] Add BAU into workflow
- [x] Fix heat_sink_T reset
- [x] Fix minor issue
- [x] Fix `heat_sink_T` estimation for flexible-moderate scenario
- [x] Add automatic copy of `custom_powerplants.csv` and `custom_busmap_48.csv` into workflow

Other:
- [x] Fix url for reuse

### CODE REVIEW

File: plots/plot_capacity_expansion.py [hunk 1]
```diff
@@ -0,0 +1,388 @@
...
+        "resistive heater",
+        "air heat pump",
+        "ground heat pump",
+        "gas boiler",
+        "biomass boiler",
+        "WWHRS",
+        "building retrofitting",
+        "WWHRS",
+     ]
+)
```
Comment (martacki) [inline plots/plot_capacity_expansion.py:164]: this seems duplicated, isn't it already included in another script?

File: plots/plot_capacity_expansion.py [hunk 2]
```diff
@@ -0,0 +1,388 @@
...
+            label = rif
+
+    for old, new in RENAME_IF_CONTAINS_DICT.items():
+        if old in label:
+            label = new
+
+    for old, new in RENAME.items():
+        if old == label:
+            label = new
+    return label
```
Comment (martacki) [inline plots/plot_capacity_expansion.py:184]: this as well

File: plots/plot_capacity_expansion.py (hunk 1 above)
Comment (yerbol-akhmetov) [reply plots/plot_capacity_expansion.py:164]: > this seems duplicated, isn't it already included in another script?

Yes, it was used in `plot_total_costs` script. I can add those constants and functions into `_helpers`. Then import them.

File: plots/plot_capacity_expansion.py (hunk 2 above)
Comment (yerbol-akhmetov) [reply plots/plot_capacity_expansion.py]: Addressed your comments.

**Pull request merged**